client/client.go

```diff
@@ -728,6 +731,16 @@ func (c *Client) SandboxController(name string) sandbox.Controller {
 	return sandboxproxy.NewSandboxController(sandboxsapi.NewControllerClient(c.conn))
 }
 
+// TransferService returns the underlying transfer service client
```

Comment:
Can you remove the client changes now? Since you aren't using it, we don't need to expand the interface for this change

Reply:
Actually I am using the `client.TransferService` to access the in-memory transfer service from CRI image service, let me know if I should do it differently.

Reply:
Looks to me like you already updated it and are doing it the right way, I see you using

```
			ts, err := ic.GetSingle(plugins.TransferPlugin)
			if err != nil {
				return nil, err
			}
```

These client functions aren't called anymore.